src/main/java/io/supertokens/oauth/Transformations.java

```diff
@@ -75,9 +68,7 @@ public static List<String> transformCookiesFromHydra(List<String> cookies) {
         for (int i = 0; i < cookies.size(); i++) {
             String cookieStr = cookies.get(i);
             if (cookieStr.startsWith("ory_hydra_")) {
-                if (cookieStr.startsWith("ory_hydra_")) {
-                    cookieStr = cookieStr.replaceFirst("ory_hydra_", "st_oauth_");
-                }
+                cookieStr = cookieStr.replaceFirst("ory_hydra_", "st_oauth_");
                 cookies.set(i, cookieStr);
```

Comment:
Are there any cookies that do not start with hydra? Do we intentionally skip them?

Reply:
no, all of them had `ory_hydra_` as prefix